nucleus/modelci/__init__.py

```diff
@@ -0,0 +1,174 @@
+"""Model CI Python Library."""
+from typing import List
+
+from nucleus.connection import Connection
+from nucleus.job import AsyncJob
+
+from .constants import (
```

Comment:
Lets only include the imports in this file, and move the client to its own folder.

So you would just add from .client import ModelCI